packages/grafana-data/src/types/featureToggles.gen.ts

```diff
@@ -747,6 +747,10 @@ export interface FeatureToggles {
   */
   newFiltersUI?: boolean;
   /**
+  * Allows access to the new react-data-grid based table component.
+  */
+  tableNG?: boolean;
```

Comment:
nit: previously anything prefixed or postfixed with `NG` would suggest Angular, is it possible to change the name of the feature toggle as we are getting rid of Angular in G12?

Reply:
I went ahead and updated the feature flag to `tableNextGen `